### PR TITLE
Adds a new brain damage line

### PR DESCRIPTION
**Pull request by Iamgoofball**
fuck

### CODE REVIEW

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -61,7 +61,8 @@
         "@pick(semicolon)N-NYAAAAAA~",
         "@pick(bug)",
         "@pick(semicolon)wtf??????????? @pick(bug)",
-        "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)"
+        "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
+		"@pick(semicolon)okay i'm reporting shitcurity don't grab prisoners with you is 1 shitcurity is 2 and asscurity is 3 = permaban i'm reporting"
```
Comment (RemieRichards) [inline strings/brain_damage_lines.json]: what was our stance on OOC in IC in code again?
I believe it was NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO.
(the 'permaban' bit)

Comment (Iamgoofball) [inline strings/brain_damage_lines.json]: there's a whole setup for randomly picking what type of "is it a bug" to pick and brain damage stuff is the ramblings of the insane and damaged

Comment (RemieRichards) [inline strings/brain_damage_lines.json]: that's a "this is a game" thing, not a "admins be spying on yo ass" thing.
plus we had that whole thing with the "bans" in descriptions of items.

**Comment by Bawhoppen** [comment]
Say it with me: "Goof's jokes AREN'T FUNNY."

**Comment by nicbn** [comment]
NO

**Comment by nicbn** [comment]
STOP ADDING FUCKING LINES

**Comment by Cyberboss** [comment]
fluff text hurts literally no one @nicbn

File: strings/brain_damage_lines.json (hunk 1 above)
Comment (Cyberboss) [inline strings/brain_damage_lines.json]: Fix indentation and remove ooc reference

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by KorPhaeron** [comment]
stale

**Pull request closed without merging**